Library/Homebrew/extend/os/linux/diagnostic.rb

```diff
@@ -59,6 +82,40 @@ def check_umask_not_zero
             echo 'umask 002' >> #{shell_profile}
         EOS
       end
+
+      def check_homebrew_prefix
+        return if HOMEBREW_PREFIX.to_s == "/home/linuxbrew/.linuxbrew"
```

Comment:
Might be nicer to use `Homebrew::DEFAULT_PREFIX` and share this logic between macOS and Linux.